src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Metadata/ModelAttributes.cs

```diff
@@ -56,7 +54,11 @@ public ModelAttributes(IEnumerable<object> propertyAttributes, IEnumerable<objec
         /// If this instance represents a parameter, the set of attributes for that parameter.
         /// Otherwise, <c>null</c>.
         /// </param>
-        public ModelAttributes(IEnumerable<object> typeAttributes, IEnumerable<object> propertyAttributes, IEnumerable<object> parameterAttributes)
+        // Reviewers: This constructor is new but not intended to be used. Should we make it internal (just for tests)?
```

Comment:
Thoughts?

Separately, this PR again uncovered code using old `ParameterBinder` overloads -- this time in tests. Should we make those `[Obsolete]` soon (please)?

Reply:
Yeah I think that's a good suggestion. Let's make this `internal`

Reply:
Discussed in person, agreed to mark the `ParameterBinder` old overloads obsolete during MQ.

Reply:
Filed #7660 about the `ParameterBinder` overloads.